charts/studio/values.yaml

```
@@ -1016,3 +1016,243 @@ serviceAccount:
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name: ""
+
+# -- Fluent Bit configuration for log collection
+fluent-bit:
+  # -- Fluent Bit enabled
+  enabled: true
```

```suggestion
  enabled: false
```

Except on DataChain clusters